CHANGES.md

```diff
@@ -80,6 +80,11 @@ To be released.
      render the favicon in terminal emulators that do not support 24-bit
      colors.  [[#168], [#282] by Hyeonseo Kim]
 
+ -  Fixed TypeScript discriminated union type narrowing issues in codegen
+    that occurred with Deno 2.4.2's stricter type checking.  Added type
+    guard functions `isNonFunctionalProperty()` and `hasSingularAccessor()`
+    to properly narrow `PropertySchema` types.  [[#303], [#305] by Lee ByeongJun]
+
```

Comment:
Since the codegen is not a user-facing component, I think we shouldn't add a changelog for this patch.